packages/data-models/src/language_model.js

```diff
@@ -452,6 +473,22 @@ class LanguageModel {
     }
   }
 
+  /**
+   * Compare two feature values with language specific logic
+   *
+   * @param {string} featureType - the feature type being compared
+   * @param {string} valueA - the first value for comparison
+   * @param {string} valueB - the second value for comparison
+   * @param {boolean} normalize - whether or not to apply normalization
+   */
+  static compareFeatureValue ( featureType, valueA, valueB, normalize = true) {
```

Comment:
Maybe add `normalize` to the options object here too? We might have other options in the future.